src/Modules/AuthModule/components/Register/Register.tsx

```diff
@@ -1,5 +1,403 @@
-import React from "react";
+import React, { useContext, useState } from "react";
+import { ErrorMessage } from "@hookform/error-message";
+import axios from "axios";
+import { useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+
+import Images from "../../../ImageModule/components/Images/Images";
+
+import BaceUrlContext from "../../../../Context/BaceUrlContext";
 
 export default function Register() {
-  return <>Register</>;
+  // instance from use navigate
+  const navigate = useNavigate();
+  // image input state
+  const [image, setImage] = useState(null);
+  // hide pass or show state
+  let [hidePassInInpt, setHidePassInInpt] = useState(true);
+  // check loading state
+  const [isLoading, setIsLoading] = useState(false);
+  // function for change password input type
+  const changePassInputType = () => {
```

Comment:
We can change this to something like `changePasswordVisibility` for more clarity and to indicate the main purpose of the function.
`changePassInputType` => doesn't tell to which type, and what we mean by pass is it really pass or password ? So all of this make it unclear to the reader.